plugins/@grouparoo/postgres/tsconfig.json

```diff
@@ -1,13 +1,14 @@
 {
   "compilerOptions": {
-    "rootDir": "./src",
+    "rootDir": ".",
```

Comment:
To allow importing packageJson

Reply:
Does this change the folder structure within the `/dist` output?  If it does, that's going to mess with how the plugin is loaded.  We expect `<pluginRoot>/dist/initializers/stuff`.  I'll bet we have this now: `<pluginRoot>/postgres/dist/initializers/stuff`

Reply:
That would explain why the build fails.

Reply:
This *might* be rectified with changes to `core/config/plugins.ts` to set a new path for the plugin roots

Reply:
This causes too many issues right now, for simply getting around a require import. Maybe worth revisiting later but reverting for now.